src/mqt/predictor/evaluation.py

```diff
@@ -184,10 +193,10 @@ def evaluate_sample_circuit(filename: str) -> dict[str, Any]:
         "num_qubits": str(Path(filename).stem).replace("_", " ").split(" ")[-1],
     }
     qc = QuantumCircuit.from_qasm_file(filename)
-    results.update(create_mqtpredictor_result(qc, "expected_fidelity", filename=filename).get_dict())
-    results.update(create_mqtpredictor_result(qc, "critical_depth", filename=filename).get_dict())
+    results.update(create_mqtpredictor_result(qc, "expected_fidelity", filename=filename, devices=devices).get_dict())
```

Comment:
Since the devices are being used in this function and also `create_mqtpredictor_result ()` I think it makes sense to call `get_available_devices()` here.